homeassistant/components/feedreader/event.py

```diff
@@ -76,8 +76,6 @@ def _async_handle_update(self) -> None:
         if content := feed_data.get("content"):
             if isinstance(content, list) and isinstance(content[0], dict):
                 content = content[0].get("value")
-        else:
-            content = feed_data.get("summary")
```

Comment:
this is not needed, since the feedparser lib does already use the summary as content in case there is no content key in the feed data